Tests/WorkflowExampleTests/Mocks/MockOrchestrationResponder.swift

```diff
@@ -10,6 +10,10 @@ import Foundation
 import Workflow
 
 class MockOrchestrationResponder: OrchestrationResponder {
+    func complete(_ workflow: AnyWorkflow, passedArgs: AnyWorkflow.PassedArgs, onFinish: ((AnyWorkflow.PassedArgs) -> Void)?) {
+        
```

Comment:
Needs to be implemented.  Probably should be copied from the other MockOrchestrationResponder.